promql/functions.go

```diff
@@ -496,6 +496,31 @@ func funcAbsent(vals []Value, args Expressions, enh *EvalNodeHelper) Vector {
 		})
 }
 
+// === absent_over_time(Vector ValueTypeMatrix) Vector ===
+func funcAbsentOverTime(vals []Value, args Expressions, enh *EvalNodeHelper) Vector {
+	if len(vals[0].(Matrix)) > 0 {
+		for _, m := range vals[0].(Matrix) {
+			if len(m.Points) > 0 {
+				return enh.out
```

Comment:
> What if storage returns a series but that series has no samples in the time range care about?
> 
> I think this is harder to get right than you'd think.

Then it is discarded here. Correct?

Reply:
No it isn't, it'll fall through and you'll end up with duplicate output series.

Reply:
I've got that case tested and fixed